patternfly/patternfly-elements
# adding ability to disable a pfe-collapse-toggle button

Fixes #516

## adding ability to disable a pfe-collapse-toggle button

### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

* added the ability to add a disabled attribute to pfe-collapse-toggle to prevent it from opening a panel

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. Visit the demo page and go to the last example at the bottom of the page. You should not be able to expand the panel.
